### ISSUE TITLE
Build problems on Windows 10

### ISSUE THREAD

**Issue by aggx** [xianyi/OpenBLAS#1294]
Hi,  I am having an issue trying to build on a windows machine (Intel/Broadwell i7-6900K @ 3.2 GHz).
I am getting a alot of warnings and dropout messages - see attached.  Seems to stop short of building the dll's I'm looking for.
[build-log.txt](https://github.com/xianyi/OpenBLAS/files/1285656/build-log.txt)
[Warnings-Log.docx](https://github.com/xianyi/OpenBLAS/files/1285658/Warnings-Log.docx)

**Comment by quickwritereader** [comment]
Hi. Could you repost log as one. Also please call `make clean` before doing make this way

` make >build.log 2>&1`
or 
` make &> build.log`

**Comment by brada4** [comment]
Identical to #658

**Comment by aggx** [comment]
Problem seems to have resolved itself!  The one thing I can point to is that I ran some updates within MSYS2 before trying the build but without restarting the app.  When I went back to rerun and collect better logs, I had shutdown and restarted the app.  Had done exactly what I had done before (make clean, etc.).  Sorry for the troubles.
[build.log.txt](https://github.com/xianyi/OpenBLAS/files/1289435/build.log.txt)
